drivers/docker/stats.go

```diff
@@ -22,6 +24,55 @@ const (
 	statsCollectorBackoffLimit = 2 * time.Minute
 )
 
+// usageSender wraps a TaskResourceUsage chan such that it supports concurrent
+// sending and closing.
+type usageSender struct {
+	closed bool
+	destCh chan<- *structs.TaskResourceUsage
+	mu     sync.Mutex
+}
+
+// newDestChPair returns a destCh that supports concurrent sending and closing,
```

Comment:
returns a `*usageSender`? its unclear what a 'destCh' is without some context I think

Reply:
Thanks. I went around and around trying to name these things sensibly and missed a spot.